modules/oe_content_organisation/modules/oe_content_organisation_person/oe_content_organisation_person.install

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Content Organisation Person installation file.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+/**
+ * Implements hook_install().
+ */
+function oe_content_organisation_person_install(): void {
+  // Add the persons entity reference field to the entity form display.
+  /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $form_display */
+  $form_display = EntityFormDisplay::load('node.oe_organisation.default');
+  if ($form_display) {
+    $oe_organisation_persons = [
+      'weight' => 29,
+      'settings' => [],
+      'third_party_settings' => [],
+      'type' => 'options_select',
```

Comment:
This should be an autocomplete I believe, all of our entity reference fields use autocomplete right?

Reply:
Indeed, I was using the autocomplete on ewcms but I'll update in here as well.